Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/Exporters/TemplatedExporter/TemplatedUnityProjectExporter.cs

```diff
@@ -52,7 +52,7 @@ public class TemplatedProjectExporter : IProjectExporter
         /// <param name="dependenciesProjectTemplatePath">Path to the dependencies project template file.</param>
         /// <param name="dependenciesPropsTemplatePath">Path to the dependencies props template file.</param>
         /// <param name="dependenciesTargetsTemplatePath">Path to the dependencies targets template file.</param>
-        public TemplatedProjectExporter(DirectoryInfo generatedOutputFolder, FileInfo solutionFileTemplatePath, FileInfo projectFileTemplatePath, FileInfo generatedProjectFileTemplatePath, FileInfo projectPropsFileTemplatePath, FileInfo projectTargetsFileTemplatePath, FileInfo msbuildForUnityCommonTemplatePath, FileInfo dependenciesProjectTemplatePath, FileInfo dependenciesPropsTemplatePath, FileInfo dependenciesTargetsTemplatePath)
+        public TemplatedUnityProjectExporter(DirectoryInfo generatedOutputFolder, FileInfo solutionFileTemplatePath, FileInfo projectFileTemplatePath, FileInfo generatedProjectFileTemplatePath, FileInfo projectPropsFileTemplatePath, FileInfo projectTargetsFileTemplatePath, FileInfo msbuildForUnityCommonTemplatePath, FileInfo dependenciesProjectTemplatePath, FileInfo dependenciesPropsTemplatePath, FileInfo dependenciesTargetsTemplatePath)
         {
```

Comment:
nit: this is really hard to read in an editor, I'd prefer if each argument had its own line.

Reply:
I may update this, but in a later change.